CrowServe/tests/CrowServeTest.cpp

```diff
@@ -25,6 +25,42 @@ TEST_CASE("CBOR de/serialization") {
 
 TEST_CASE("CrowServe integration") {
     CrowServe::Socket crow_serve;
-    REQUIRE(crow_serve.try_init());
-    REQUIRE(crow_serve.receive());
+    crow_serve.listen(CrowServe::Overloaded{
+        [](const CrownLink::ConnectionRequest &message) {
```

Comment:
We can remove the overloaded part, like I posted in one of the examples. Tbh this looks like a decent api though! Nice work!

Reply:
Reference alignment is wrong, again something for formatter ;)